tests/kat_sig_stfl.c

```diff
@@ -297,7 +298,44 @@ OQS_STATUS sig_stfl_kat(const char *method_name, const char *katfile) {
 
 	ret = OQS_SUCCESS;
 	goto cleanup;
+#else
+	/*
+	 * Signature generation is disabled so only signature verification can be tested.
```

Comment:
I know that we don't really have any "unhappy path" tests in liboqs, but I think they would be useful here as a sanity check to make sure that calling keygen / sign does, in fact, result in an error when those operations are disabled in the config.

Reply:
Thanks. Will do this on another test since the KAT test compares hashes of the expected output to determine success status of the test.

Reply:
Incorporated some of this in test_sig_stfl.c.